final_prep/annotations.rb

```diff
@@ -4,10 +4,15 @@
 
 # Build a Bear
 
+# Create method (function) to build a bear with 5 arguments
```

Comment:
These are called _parameters_, not arguments, when you define them in the method. Arguments and parameters are 2 sides of the same coin, so to speak.